### DISCUSSION TITLE
dark mode?

### DISCUSSION THREAD

**Discussion by bluelayer** [tnodir/fort#61] [Q&A] [answered]
hello!

today I found my FF running in "white mode", I have all settings in dark mode, it's just me?

Thanks!

**Comment by tnodir** [comment, marked as answer]
Hello.

1. Fort Firewall's 32-bit version does not support Dark mode.

Maybe you installed the " FortFirewall-*-windows-x86.exe "?

2. Or you are running it as Admin.

Fort Firewall’s UI retrieves theme settings (dark/light) of current running user.

In most cases Admin user is not configured to use dark mode theme.

**Reply by bluelayer** [reply]
I don't know how but I have 32b installed =)

**Comment by bluelayer** [comment]
well... I think I found the culprit! when you open FF options panel AND you have one update, you click on the link and the version that is downloaded is the x86 version... then I just clicked next next next =)

so... BUG?

**Reply by tnodir** [reply]
Yes, let me fix it..

**Reply by tnodir** [reply]
Fixed in [v3.8.6](https://github.com/tnodir/fort/releases/tag/v3.8.6).
